docs/services/data/_index.md

```diff
@@ -43,41 +43,47 @@ Or learn more about Viam's data management features, which can be broken down in
 
 ## On-device data management
 
-Manage data directly on your robot, with configurable data capture from various components, automated recovery after interruptions, and secure cloud synchronization for efficient data storage and management.
+The data management service:
+
+- Automatically resumes capturing data after your machine restarts.
+- Securely transfers data to the cloud at the frequency you define.
+- Automatically deletes local data after syncing for space management.
 
 {{< cards >}}
-{{% manualcard link="/services/data/capture/" %}}
+{{% manualcard link="/services/data/capture-sync/" %}}
 
-### Data capture
+### Data capture and cloud sync
 
-Configurable for various robot components like cameras, sensors, and motors, with configurable capture frequency.
-Automatically resumes after robot restarts.
+More details and how to configure capture and sync.
 
 {{% /manualcard %}}
-{{% manualcard link="/services/data/cloud-sync/" %}}
-
-### Cloud sync
+{{< /cards >}}
 
-Securely transfer data to the cloud at the frequency you define.
-Resilient to interruptions, and deletes local data post-sync for space management.
+You can configure the frequency of data capture individually for each supported component:
```

Comment:
you can also do this for the vision service. I think let's either move these cards into https://docs-test.viam.dev/3473/services/data/capture-sync/#supported-components-and-services or just rely on that to display what's supported.

Reply:
I thought this was a good way to make things more visual and concrete. Can we assume they'll go to the other page?

Reply:
Ok. Then we'd need to add vision service here too and make it more general. Or is there a reason you think we should point out components but not services?
I do think we may want to move away from placing focus on the configuration and focus more on the capturing aspect -> "You can capture data from most components and services." They'll dig into the rest if they want to use it

Reply:
```suggestion
You can capture data from most components and services:
```

Reply:
```suggestion
You can capture data from any of the following components and services:
```